cmd/stop.go

```diff
@@ -30,13 +33,39 @@ var StopCmd = &cobra.Command{
 	Example: `
 # Stop Dapr application
 dapr stop --app-id <ID>
+
+# Stop multiple apps by providing a run config file
+dapr stop --run-file dapr.yaml
+
+# Stop multiple apps by providing a directory path containing the run config file(dapr.yaml)
+dapr stop --run-file /path/to/directory
 `,
 	Run: func(cmd *cobra.Command, args []string) {
+		var err error
+		if len(runFilePath) > 0 {
+			if runtime.GOOS == string(WINDOWS) {
+				print.FailureStatusEvent(os.Stderr, "The stop command with run file is not supported on Windows")
+				os.Exit(1)
+			}
+			runFilePath, err = getRunFilePath(runFilePath)
+			if err != nil {
+				print.FailureStatusEvent(os.Stderr, "Failed to get run file path: %v", err)
+				os.Exit(1)
+			}
+			err = executeStopWithRunFile(runFilePath)
+			if err != nil {
+				print.FailureStatusEvent(os.Stderr, "Failed to stop Dapr and app processes: %s", err)
+			} else {
+				print.SuccessStatusEvent(os.Stdout, "Dapr and app processes stopped successfully")
+			}
+			return
+		}
 		if stopAppID != "" {
 			args = append(args, stopAppID)
 		}
+		cliPIDToNoOfApps := standalone.GetCLiPIDCountMap()
```

Comment:
Why not get this map directly in `standalone.stop` instead of `cmd/stop`?

Reply:
I placed here to handle the scenario like this `dapr stop app1 app2 app3`. We need to pass the original count. Relying on list command to return updated values will not work as it prcesses takes some time to get deleted.